mkdocs.yml

```
@@ -0,0 +1,47 @@
+  - literate-nav:
+      nav_file: SUMMARY.md
+
+markdown_extensions:
+    - markdown_include.include:
+        base_path: docs
+    - admonition
+    - toc:
+        permalink: ""
+
+extra_css:
+    - css/custom.css
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 2 but found 4

(indentation)